_studio/mfx_lib/encode_hw/shared/ehw_task_manager.cpp

```diff
@@ -139,6 +139,10 @@ mfxStatus TaskManager::TaskQuery(StorageW& inTask)
     std::unique_lock<std::mutex> closeGuard(m_closeMtx);
     auto pBs = GetBS(inTask);
     MFX_CHECK(pBs, MFX_ERR_NONE);
+    if (m_bSync)
+    {
+        m_bSync = false; return MFX_ERR_NONE;
```

Comment:
move return to new line, add comment why we need to skip query stage for this case.

Reply:
Fixed